lib/OpenQA/WebAPI.pm

```diff
@@ -249,62 +249,56 @@ sub startup {
         }
     }
 
-    $self->plugin(bootstrap3 => {css => [], js => []});
-
-    $self->asset(
-        'step_edit.js' => qw(/javascripts/needleedit.js
-          /javascripts/needleeditor.js
-          /javascripts/shapes.js
-          /javascripts/keyevent.js/)
+    $self->asset->process(
+        'step_edit.js' => qw(javascripts/needleedit.js
+          javascripts/needleeditor.js
+          javascripts/shapes.js
+          javascripts/keyevent.js/)
     );
 
-    my @js = qw(/javascripts/jquery-1.11.2.js
-      /javascripts/jquery_ujs.js
-      /javascripts/chosen.jquery.js
-      /javascripts/openqa.js
-      /javascripts/jquery.dataTables.js
-      /javascripts/admintable.js
-      /javascripts/admin_user.js
-      /javascripts/admin_needle.js
-      /javascripts/audit_log.js
-      /javascripts/jquery.timeago.js
-      /javascripts/tests.js
-      /javascripts/assets.js
-      /javascripts/job_templates.js
-      /javascripts/overview.js
-      /javascripts/comments.js);
-    my @css = qw(/stylesheets/font-awesome.css
-      /stylesheets/chosen.css
-      /stylesheets/overview.scss
-      /stylesheets/comments.css
-      /stylesheets/openqa.css );
-
-    # preprocessors to expend the url() definitions in the css
-    $self->asset->preprocessors->add(
-        css => sub {
-            my ($assetpack, $text, $file) = @_;
-            $$text =~ s!url\('!url('../images/!g if $file =~ /chosen.css/;
-        });
-
-    $self->asset('app.css' => (qw(/stylesheets/jquery.dataTables.css /stylesheets/tables.css), @css));
-    $self->asset('app.js' => @js);
-    my $path = Mojolicious::Plugin::Bootstrap3->asset_path('sass');
-    $ENV{SASS_PATH} = ".:$path";
-    $self->asset(
-        'bootstrap.css' => (
-            qw(/sass/bentostrap.scss
-              /stylesheets/dataTables.bootstrap.css
-              ), @css
-        ));
-    $self->asset(
+    my @js = qw(javascripts/jquery-1.11.2.js
+      javascripts/jquery_ujs.js
+      javascripts/chosen.jquery.js
+      javascripts/openqa.js
+      javascripts/jquery.dataTables.js
+      javascripts/admintable.js
+      javascripts/admin_user.js
+      javascripts/admin_needle.js
+      javascripts/audit_log.js
+      javascripts/jquery.timeago.js
+      javascripts/tests.js
+      javascripts/assets.js
+      javascripts/job_templates.js
+      javascripts/overview.js
+      javascripts/comments.js);
+    my @css = qw(stylesheets/font-awesome.css
+      stylesheets/chosen.css
+      stylesheets/overview.scss
+      stylesheets/comments.css
+      stylesheets/openqa.css );
+
+    # preprocessors to expend the url() definitions in the css - TODO: reenable chosen in job group
```

Comment:
yeah, I've already  seen that documentation. But as we don't use this plugin atm, it's work for later